### DISCUSSION TITLE
Controllo Ezviz

### DISCUSSION THREAD

**Discussion by Moxser** [indomus/forum#288] [HUB: Home Assistant]
Ciao a tutti, ho notato che è finalmente tornata la possibilità di integrare le telecamere della Ezviz su Home assistant. Dopo aver integrato le mie 5 ip cam ora ho diversi sensori e switch ma la cosa che mi lascia un po perplesso è il loro utilizzo. Ad esempio ho creato un automazione che quando torno mi attivi la modalità sleep, e fin qui tutto perfetto, poi ne ho creata una che quando esco disattivi la stessa modalità. La cosa strana è che quando attivo la modalità sleep posso riattivare le telecamere solo dall'applicazione della Ezviz e non da HA. Qualcuno ha qualche suggerimento in merito?

**Comment by bf69** [comment]
utilizzo lo stesso sistema e a me funziona, ho notato che i segnali sono un po ritardati, nel senso che per esempio al cambio di stato dello switch sleep a video mi ritorna allo stato precedente per un paio di secondi, poi probabilmente si riaggiorna con il cloud ezviz

**Reply by bf69** [reply]
non ho trovato nessun altro modo per attivare il motion detect...

**Comment by Moxser** [comment]
Io sinceramente se inserisco sleep da HA o da app Ezviz dopo pochi minuti da HA le cam non sono più attivabili. Sembra non ci sia comunicazione bilaterale tra cloud e HA.
